components/browser/menu2/README.md

```diff
@@ -0,0 +1,44 @@
+# [Android Components](../../../README.md) > Browser > Menu 2
+
+A generic menu with customizable items primarily for browser toolbars.
+
+This replaces the [browser-menu](../menu) component.
+
+## Usage
+
+### Setting up the dependency
+
+Use Gradle to download the library from [maven.mozilla.org](https://maven.mozilla.org/) ([Setup repository](../../../README.md#maven-repository)):
+
+```Groovy
+implementation "org.mozilla.components:browser-menu2:{latest-version}"
+```
+
+### MenuController
+Sample code can be found in [Sample Toolbar app](https://github.com/mozilla-mobile/android-components/tree/master/samples/toolbar).
+
+There are multiple properties that you customize of the menu browser by just adding them into your dimens.xml file.
```

Comment:
Will this still be true?  Not that I would expect these to change all that often, but it might be more internally consistent to include these at the top level of passed state (with sensible defaults).

Reply:
This will still be true, although the dimens mentioned have the old names (new one have menu2, not menu)